Sources/ApiVideoPlayer/ApiVideoPlayerInformationNowPlaying.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import MediaPlayer
+
+protocol InformationNowPlaying {
+    func update(metadata: [String: Any]?)
+    func pause(currentTime: CMTime)
+    func play(currentTime: CMTime)
+    func overrideInformations(for key: String, value: Any)
+}
+
+#if !os(macOS)
+class ApiVideoPlayerInformationNowPlaying: InformationNowPlaying {
+    private var infos = [String: Any]()
+    private let taskExecutor: TasksExecutorProtocol.Type
+
+    init(taskExecutor: TasksExecutorProtocol.Type) {
+        self.taskExecutor = taskExecutor
+    }
+
+    func update(metadata: [String: Any]?) {
+        infos[MPMediaItemPropertyTitle] = metadata?["title"] ?? ""
+        if let duration = metadata?["duration"] as? CMTime {
+            print("duration info center : \(duration)")
+            infos[MPMediaItemPropertyPlaybackDuration] = duration.seconds
+        }
+        if let currentTime = metadata?["currentTime"] as? CMTime {
+            infos[MPNowPlayingInfoPropertyElapsedPlaybackTime] = round(currentTime.seconds)
+        }
+        if let isLive = metadata?["isLive"] as? Bool {
+            infos[MPNowPlayingInfoPropertyIsLiveStream] = isLive
+        }
+        infos[MPNowPlayingInfoPropertyPlaybackRate] = 1.0
+
+        if let imageStr = metadata?["thumbnailUrl"] as? String {
+            if let url = URL(string: imageStr) {
+                updateRemoteImage(url: url)
+            }
+        }
+        MPNowPlayingInfoCenter.default().nowPlayingInfo = infos
```

Comment:
Shoudnot `MPNowPlayingInfoCenter.default().nowPlayingInfo` set to `nil` when player is stopped or destroyed?